### PR TITLE
bump chainlink-common

### PR DESCRIPTION
**Pull request by 0xnogo**

### CODE REVIEW

File: internal/reader/ccip.go [hunk 1]
```diff
@@ -412,6 +411,7 @@ func (r *CCIPChainReader) getSourceChainsConfig(
 				ctx,
 				consts.ContractNameOffRamp,
 				consts.MethodNameGetSourceChainConfig,
+				primitives.Finalized,
```
Comment (0xnogo) [inline internal/reader/ccip.go:414]: New input param introduced. It represents the level of confidence of the data asked

Comment (makramkd) [reply internal/reader/ccip.go:414]: Interesting. I don't think this particular call needs to be finalized though

File: internal/reader/home_chain.go [hunk 2]
```diff
@@ -116,7 +117,12 @@ func (r *homeChainPoller) poll() {
 func (r *homeChainPoller) fetchAndSetConfigs(ctx context.Context) error {
 	var chainConfigInfos []ChainConfigInfo
 	err := r.homeChainReader.GetLatestValue(
-		ctx, consts.ContractNameCCIPConfig, consts.MethodNameGetAllChainConfigs, nil, &chainConfigInfos,
+		ctx,
+		consts.ContractNameCCIPConfig,
+		consts.MethodNameGetAllChainConfigs,
+		primitives.Finalized,
```
Comment (makramkd) [inline internal/reader/home_chain.go:123]: Ditto here, what other options are there?

Comment (0xnogo) [reply internal/reader/home_chain.go:123]: Finalized or Unconfirmed. Why would we want an unfinalized data?

File: internal/reader/ccip.go (hunk 1 above)
Comment (makramkd) [reply internal/reader/ccip.go:414]: Doesn't need to be finalized.

File: internal/reader/home_chain.go (hunk 2 above)
Comment (makramkd) [reply internal/reader/home_chain.go:123]: Does not need to be finalized, can use Unconfirmed.

File: internal/reader/onchain_prices_reader.go [hunk 3]
```diff
@@ -54,7 +55,14 @@ func (pr *OnchainTokenPricesReader) GetTokenPricesUSD(
 			if staticPrice, exists := pr.TokenPriceConfig.StaticPrices[token]; exists {
 				price.Set(&staticPrice)
 			} else {
-				if err := pr.ContractReader.GetLatestValue(ctx, contractName, functionName, token, price); err != nil {
+				if err :=
+					pr.ContractReader.GetLatestValue(
+						ctx,
+						contractName,
+						functionName,
+						primitives.Finalized,
```
Comment (makramkd) [inline internal/reader/onchain_prices_reader.go:63]: Does not need to be finalized.

**Review by makramkd — APPROVED** [review]

File: internal/reader/onchain_prices_reader.go [hunk 4]
```diff
@@ -54,7 +55,14 @@ func (pr *OnchainTokenPricesReader) GetTokenPricesUSD(
 			if staticPrice, exists := pr.TokenPriceConfig.StaticPrices[token]; exists {
 				price.Set(&staticPrice)
 			} else {
-				if err := pr.ContractReader.GetLatestValue(ctx, contractName, functionName, token, price); err != nil {
+				if err :=
```
Comment (winder) [inline internal/reader/onchain_prices_reader.go:58]: style nit: I'm personally not a fan of "one liner" if conditions when they span multiple lines.

**Review by winder — APPROVED** [review]
Changes look fine. Just had a minor suggestion for you to consider.

**Review by asoliman92 — APPROVED** [review]

**Pull request merged**